### ISSUE TITLE
SPIKE: Deal ticket validation UX

### ISSUE THREAD

**Issue by bwallacee** [vegaprotocol/frontend-monorepo#4067]
## Story

As a user of console
I want to be informed / blocked / updated on different validations that exist on the deal ticket as I am placing orders that would fail on transaction
So that I can find out sooner in the process that the order I am trying to make is invalid.

## Acceptance Criteria

- Break down the order types into section tickets
- Work out all of the different validations that exist
- Work out our preffered UX for validations (e.g. do we block users, do we force update changes, do we inform the user?
- Ensure that we are only providing feedback not preventing submitting an order

## Tasks

- [ ] UX 
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted

## Additional details / background info
Basically there are a lot of different validations that exist on the deal ticket, rather than letting a user proceed with the order then the Backend failing the transaction, the UI should have validation /update (hide non valid options etc.) based on options chosen.

**Comment by JonRay15** [comment]
This will never happen .... are you happy to close @bwallacee ?

**Comment by JonRay15** [comment]
I cant see any way we prioritise this sort of review at the moment so closing.
